src/ExecuteSelectorCondition.sol

```diff
@@ -82,7 +87,9 @@ contract ExecuteSelectorCondition is
             (bytes32, Action[], uint256)
         );
         for (uint256 i; i < _actions.length; i++) {
-            if (
+            if (_actions[i].data.length != 0 && _actions[i].data.length < 4) {
```

Comment:
As above, having reserved transfer would eliminate the need to check here

Reply:
Also, we don't check anywhere the value sent inside IDAO.action.

IMO might be a nice validation to check that if you're doing a native transfer, you are sending native. 

Otherwise it's a zero value transfer and I wonder why we would support such a thing